idaes/gas_solid_contactors/flowsheets/dyn_TGA_example.py

```diff
@@ -0,0 +1,100 @@
+#################################################################################
+# The Institute for the Design of Advanced Energy Systems Integrated Platform
+# Framework (IDAES IP) was produced under the DOE Institute for the
+# Design of Advanced Energy Systems (IDAES), and is copyright (c) 2018-2021
+# by the software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia University
+# Research Corporation, et al.  All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and
+# license information.
+#################################################################################
+"""
+Flowsheet example of the 2-region bubbling fluidized bed (BFB) model
+for a methane combustion with iron-oxide case study
```

Comment:
Change this description to something on the model i.e. "flowsheet example of the 0DFixedBed model for an iron-oxide reduction with methane case study."